pyccl/__init__.py

```diff
@@ -60,6 +57,26 @@
 # Specific to massive neutrinos
 from .neutrinos import Omeganuh2, nu_masses
 
+# Halo concentration
+from .concentration import (
+    concentration_duffy08_200mat,
+    concentration_duffy08_200crit,
+    concentration_bhattacharya11_200mat,
+    concentration_bhattacharya11_200crit)
+
+# Halo mass definitions
+from .massdef import mass2radius_lagrangian, HMDef, HMDef200mat, HMDef200crit
+from .hmfunc import (
+    sigmaM, MassFunc, MassFuncPress74,
+    MassFuncSheth99, MassFuncJenkins01,
+    MassFuncTinker08, MassFuncTinker10,
+    MassFuncWatson13, MassFuncAngulo12,
+    MassFuncDespali16, MassFuncBocquet16)
+from .hbias import (
+    HBiasFunc, HBiasFuncSheth99,
+    HBiasFuncSheth01, HBiasFuncTinker10,
+    HBiasFuncBhattacharya11)
+
```

Comment:
We are adding a lot to the global namespace here. Maybe we should keep this stuff in some other module, like `pyccl.halos` or something?

Reply:
Also HBias -> HaloBias. Three letters don't matter when you need to type Tinker etc.

Reply:
I've put all of this into a `halos` module, and renamed a few things to make it more transparent.

Reply:
yay!